.build/runbuild.ps1

```diff
@@ -20,7 +20,7 @@ properties {
     [string]$minimumSdkVersion     = "8.0.100"
 
     #test parameters
-    [string]$testPlatforms         = "x64"
+    [string]$testPlatforms         = ""
```

Comment:
I'm not sure what you mean, this does still allow for the parameter to be passed in. It just defaults it to your current architecture if not provided (further below with the IsNullOrWhiteSpace check). That's why this was changed to blank, so that we can detect if it was not provided.

Reply:
Actually, if we did have a new parameter on `build.ps1`, it should be named `--test-platform` (maybe `-tp` for a short name) because the command is `build` and this only applies to running the tests.